src/modules/PackageSlideShow.rb

```diff
@@ -118,72 +114,46 @@ def GetPackageSummary
     # **************  Formatting functions and helpers **************************
     # ***************************************************************************
 
-    # Sum up all list items
+    # Sum up all list items. Flatten list.
     #
-    # @param sizes [Array<Fixnum>] Sizes to sum
+    # @param sizes [Array<Fixnum|Array>] Sizes to sum
     # @return [Fixnum] Sizes sum
     def ListSum(sizes)
-      sizes.reduce(0) { |a, e| (e == -1) ? a : a + e }
-    end
-
-    # Sum up all positive list items, but cut off individual items at a maximum value.
-    # Negative return values indicate overflow of any individual item at "max_cutoff".
-    # In this case, the absolute value of the return value is "max_cutoff" * number of overflows.
-    # (e.g. >2hour + >2hours + 1:13:20 => >4hours
-    #
-    def ListSumCutOff(sizes, max_cutoff)
-      sizes = deep_copy(sizes)
-      overflow = 0
-      sum = 0
-
-      Builtins.foreach(sizes) do |item|
-        if Ops.greater_than(item, 0)
-          if Ops.greater_than(item, max_cutoff)
-            overflow = Ops.add(overflow, 1)
-          else
-            sum = Ops.add(sum, item)
-          end
-        end
-      end
-
-      sum = Ops.multiply(Ops.unary_minus(overflow), max_cutoff) if Ops.greater_than(overflow, 0)
-
-      sum
+      sizes.flatten.reduce(0) { |a, e| (e < 0) ? a : a + e }
```

Comment:
A comment would be nice how this is working :-)

Reply:
yeah, I should also improve method documentation that it sum only positive values.
Do you think that this code will be more readable? `sizes.flatten.select(:positive?).reduce(0, :+)`

Reply:
nice :-)